packages/material-ui/src/Slider/Slider.test.js

```diff
@@ -133,7 +120,7 @@ describe('<Slider />', () => {
   describe('prop: orientation', () => {
     it('should render with the vertical classes', () => {
       const { container, getByRole } = render(<Slider orientation="vertical" value={0} />);
-      expect(container.firstChild).to.have.class(classes.vertical);
+      expect(container.firstChild).to.have.class('MuiSlider-vertical');
```

Comment:
What if the component was to export a `classes` object? So it could be used in the tests, and by developers who want more type safety?
```suggestion
      expect(container.firstChild).to.have.class(classes.vertical);
```

Reply:
Done, with https://github.com/mui-org/material-ui/pull/23308/commits/db20a1c882b2e741efc2c710173a36a9a081a466 let me know if this is what you had in mind